src/controllers/developers/developers.js

```diff
@@ -62,3 +58,28 @@ export const postDeveloper = async (req, res, next) => {
     return next(err);
   }
 };
+
+export const getDeveloper = async (req, res, next) => {
+  const connection = createDbConnection();
+  const Developer = connection.model('Developer', developerSchema);
+  const { id } = req.params;
+  try {
+    const developer = await Developer.findById(id);
+
+    if (!developer) {
+      throw new Error("Developer doesn't exist");
```

Comment:
nit: please use single quotes instead of double quotes
```suggestion
      throw new Error('Developer doesn\'t exist');
```